envoy/network/connection.h

```diff
@@ -255,7 +255,7 @@ class Connection : public Event::DeferredDeletable,
    * are installed.
    * @param data Supplies the data to write to the connection.
    * @param end_stream If true, this indicates that this is the last write to the connection. If
-   *        end_stream is true, the connection is half-closed. This may only be set to true if
```

Comment:
I think the comment was correct as is.
If the stream is ended (end stream = true) the connection is half closed.
The connection is fully closed if the read side is half closed and the write side is half closed.
/wait

Reply:
sorry my bad.